src/components/organisms/AssetActions/Pool/Remove.tsx

```diff
@@ -220,21 +251,40 @@ export default function Remove({
           </Button>
         </div>
       </form>
-
       <div className={styles.output}>
         <div>
           <p>{content.output.titleIn}</p>
-          <Token symbol="pool shares" balance={amountPoolShares} noIcon />
+          {isAdvanced ? (
```

Comment:
was there a specific reason to replace this with a condition here? Cause both return the exact same component with the exact same props

Reply:
ah, I remember, this is just some leftover from the gazillion refactors we went through in here, so should be good to just restore what we had originally

Reply:
I forgot to remove that condition when I replaced the maxShares with the amountPoolShares, in the simple case. I correct that immediately.